### DISCUSSION TITLE
NEOGEO Games in FBA Issue

### DISCUSSION THREAD

**Discussion by zugurudumba** [eduardofilo/RG350_adam_image#164] [Q&A] [answered]
Hi,

NEOGEO games look like this when running on FBA on RG350M. Any idea what's going on?

Thanks.
![PXL_20220116_143638625](https://user-images.githubusercontent.com/96993111/149665055-bd294d9b-4048-43dc-9a44-0bfd547b6fe2.jpg)

**Comment by eduardofilo** [comment, marked as answer]
Yes, it is an old problem. It can be fixed disabling hardware scaling in FBA UX (the adjustment is preserved when the games are later opened from SimpleMenu) or installing this FBA release that will be installed in Adam 1.4:

https://github.com/plrguez/fba-sdl/releases/tag/r19_2021-10-18-.44_od_beta-fix-06

**Comment by zugurudumba** [comment]
Awesome, it works now. Thanks for the tip.
